### PR TITLE
Add nanosvg package

### PR DESCRIPTION
**Pull request by bkotzz**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/create/cmake.html)
  step by step carefully. **[Yes]**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/bkotzz/hunter/builds/20852865
  * https://travis-ci.com/bkotzz/hunter/builds/94033477

* I have submitted CI configs to https://github.com/ingenue/hunter targeting `pkg.template` branch,
  see this merged pull request https://github.com/ingenue/hunter/pull/304

* My change will resolve this "New package" request: https://github.com/ruslo/hunter/issues/1651

---

### CODE REVIEW

File: cmake/projects/nanosvg/hunter.cmake [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+ hunter_add_version(
+    PACKAGE_NAME
+    nanosvg
+    VERSION
+    "0.0.0-p0"
```
Comment (ruslo) [inline cmake/projects/nanosvg/hunter.cmake]: Please keep it consistent with git tag, should be `0.0.0-2b08dee-p0`.

File: cmake/projects/nanosvg/hunter.cmake [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    URL
+    "https://github.com/hunter-packages/nanosvg/archive/v0.0.0-2b08dee-p0.tar.gz"
+    SHA1
+    ee4b86de4478fc912510bada2341d4f8faa8feaf
+)
+
+hunter_pick_scheme(DEFAULT url_sha1_cmake)
+hunter_cacheable(nanosvg)
+hunter_download(PACKAGE_NAME nanosvg)
```
Comment (ruslo) [inline cmake/projects/nanosvg/hunter.cmake]: https://0.readthedocs.io/en/latest/git.html#check-there-is-a-new-line-at-end-of-file

File: examples/nanosvg/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+include("../common.cmake")
+
+project(download-nanosvg)
+# DOCUMENTATION_START {
+hunter_add_package(nanosvg)
+find_package(nanosvg CONFIG REQUIRED)
+
+add_executable(main main.cpp)
+target_link_libraries(main PUBLIC nanosvg::nanosvg)
+# DOCUMENTATION_END }
```
Comment (ruslo) [inline examples/nanosvg/CMakeLists.txt]: https://0.readthedocs.io/en/latest/git.html#check-there-is-a-new-line-at-end-of-file

File: docs/packages/pkg/nanosvg.rst [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+nanosvg
+=======
+
+-  `Official <https://github.com/memononen/nanosvg>`__
+-  `Hunterized <https://github.com/hunter-packages/nanosvg>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/nanosvg/CMakeLists.txt>`__
+-  Added by `Brad Kotsopoulos <https://github.com/bkotzz>`__ (`pr-1658 <https://github.com/ruslo/hunter/pull/1658>`__)
+
+.. literalinclude:: /../examples/PNG/CMakeLists.txt
```
[the remark this answers is not in the document]
Comment (bkotzz) [reply docs/packages/pkg/nanosvg.rst]: Whoops, thanks for catching

**Comment by ruslo** [comment]
Uploading:
* https://travis-ci.org/ingenue/hunter/builds/466244126?utm_source=github_status&utm_medium=notification
* https://ci.appveyor.com/project/ingenue/hunter/builds/20912165

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.23.69

**Pull request merged**